src/state/routing/slice.ts

```diff
@@ -36,7 +37,7 @@ const DEFAULT_QUERY_PARAMS = {
 export const routingApi = createApi({
   reducerPath: 'routingApi',
   baseQuery: fetchBaseQuery({
-    baseUrl: 'https://api.uniswap.org/v1/',
+    baseUrl: 'https://beta.api.uniswap.org/v1/',
```

Comment:
@zhongeric this should still point to prod. we will test on beta this morning and deploy to prod before releasing interface

Reply:
ok sg